Neos.ContentRepository.LegacyNodeMigration/Classes/NodeDataToAssetsProcessor.php

```diff
@@ -62,7 +62,12 @@ public function run(): ProcessorResult
                 continue;
             }
             foreach ($properties as $propertyName => $propertyValue) {
-                $propertyType = $nodeType->getPropertyType($propertyName);
+                try {
+                    $propertyType = $nodeType->getPropertyType($propertyName);
+                } catch (\InvalidArgumentException $exception) {
```

Comment:
I don't see a corresponding `throw new \InvalidArgumentException` code in [NodeType::getPropertyType()](https://github.com/markusguenther/neos-development-collection/blob/731fc7fb8fe1d922a83b53218e948c51e206cca7/Neos.ContentRepository/Classes/Domain/Model/NodeType.php#L436) ?

Reply:
=> https://github.com/neos/neos-development-collection/blob/9.0/Neos.ContentRepository.Core/Classes/NodeType/NodeType.php#L422-L425